bedrock/mozorg/redirects.py

```diff
@@ -743,4 +743,7 @@ def decider(request, **kwargs):
 
     # Issue 9984
     redirect(r'^/about/legal/fraud-report/?$', '/about/legal/defend-mozilla-trademarks/'),
+    
```

Comment:
The CI tests failed due to some white space in this blank line. We use the `flake8` tool to verify some style things, and this is one of them. I have my editor set to auto remove trailing whitespace to avoid this kind of thing.